CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Before you contribute to the SPDX License List
+Please familiarize yourself with the SPDX License List and its supporting documentation, so that you understand the pertinent context around the list itself:
+* [Overview of the SPDX License List](https://spdx.org/spdx-license-list/license-list-overview) explains some key background, principles for the inclusion of a license on the SPDX License List, as well as an explanation of the fields contained on the list.
+* [SPDX License List Matching Guidelines](https://spdx.org/spdx-license-list/matching-guidelines) provides guidelines to be used for the purposes of matching licenses and license exceptions against those included on the SPDX License List. 
+
+# How to contribute/participate: general
+The SPDX License List is maintained by the SPDX Legal Team. Work for the SPDX Legal Team is done both via the mailing list and bi-weekly calls. In particular, the bi-weekly calls are used to discuss topics and issues that may be difficult to only discuss via email.
+You can join the mailing list and otherwise manage your subscription [via spdx-legal mailman](https://lists.spdx.org/mailman/listinfo/spdx-legal). Information on the bi-weekly calls and the working area for the SPDX Legal Team, can be found on the [SPDX wiki](https://wiki.spdx.org/view/Legal_Team)
+
+## Request a new license or exception be added to the SPDX License List
+The SPDX Legal Team greatly appreciates proposals for new open source licenses or exceptions to be added to the SPDX License List.  To request that a license or exception be added to the SPDX License List, please:
+
+1)  First, review the [license list overview and inclusion principles](https://spdx.org/spdx-license-list/license-list-overview).
```

Comment:
I don't think we need “First” within an enumerated list.  And in canonical Markdown, [ordered lists use numbers followed by periods][1].  So I think this should be:

1. Review the …

The link to `license-list-overview` is already in the opening section here, so maybe we just want to drop this list entry entirely?

[1]: https://daringfireball.net/projects/markdown/syntax#list

Reply:
given how obvious it is (and often) that people don't read this info, I think repeating it isn't so bad ;)